sproutsocial/seeds-packets
# [BCT-1177] Add new offset utility

## Details of Change

- New utilities for grid offsets. (1-3 columns, left, right, horizontal)
- Going to be used in the new nav, but these will be helpful everywhere.

## Relevant Links

- [Jira Card](https://sprout.atlassian.net/browse/BCT-1177)
- [Test Site](https://cheech-test.stagely.sproutsocial.com/offset-test/)

## Steps to Test or Reproduce

- Go to the test box linked above and verify the column offsets are correct. 
- **Protip**: You can use [this bookmarklet](https://hastebin.int.sproutsocial.com/juxuxosudo.groovy) to toggle our grid.

![cheech-test stagely sproutsocial com_offset-test_(Laptop with MDPI screen)](https://user-images.githubusercontent.com/5562514/76460350-da39cd00-63ab-11ea-956c-4697c646cbcd.png)
